src/include/main/client_context.h

```diff
@@ -7,13 +7,21 @@
 namespace kuzu {
 namespace main {
 
-// Contain client side configuration.
-// We make profiler associated per query, so profiler is not maintained in client context.
+/**
```

Comment:
I don't think we need to add comments in this file. This is not supposed to be a file for public APIs.